### PR TITLE
Release notes/changelog 0.8.4

### PR DESCRIPTION
**Pull request by MadelineMurray**
Added release notes/changelog for v0.8.4

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,110 @@
...
+The container can be running or stopped when you copy the key and data directory. If your node was 
+fully synchronized to MainNet, the data directory will be ~2TB.  
+
+When restarting your node with the v0.8.4 Docker image:
+
+* Save the node key in the [`key` file](https://docs.pantheon.pegasys.tech/en/latest/Configuring-Pantheon/Node-Keys/#node-private-key) in the data 
+    directory or specify the location using the [`--node-private-key` option](https://docs.pantheon.pegasys.tech/en/latest/Configuring-Pantheon/Node-Keys/#specifying-a-custom-node-private-key-file).  
+* Specify the `<destination_directory` as a [volume for the data directory](https://docs.pantheon.pegasys.tech/en/latest/Getting-Started/Run-Docker-Image/#data-directory). 
+
+### Upcoming Changes in v0.9
```
Comment (NicolasMassart) [inline CHANGELOG.md]: Shouldn't we put upcoming changes in a roadmap or something else ? I don't feel comfortable with having upcoming changes in a changelog. I feel that it should only list already made changes. It's just that I never saw that in other changelogs.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,110 @@
...
+In v0.9, the Private Network Quickstart will be moved to a separate repository and the existing Private Network 
+Quickstart removed from the Pantheon repository. The [Private Network Quickstart tutorial](https://docs.pantheon.pegasys.tech/en/latest/Getting-Started/Private-Network-Quickstart/) 
+will be updated and use the Private Network Quickstart in the separate repository.
+
+### Bug Fixes
+- Fixing default resource locations inside docker [\#529](https://github.com/PegaSysEng/pantheon/pull/529)
+- NewRoundMessageValidator ignores Round Number when comparing blocks [\#523](https://github.com/PegaSysEng/pantheon/pull/523) 
+- Fix Array Configurable command line options [\#514](https://github.com/PegaSysEng/pantheon/pull/514) 
+
+## Additions and Improvements
```
Comment (NicolasMassart) [inline CHANGELOG.md]: What's the real difference between "improvements" and "technical improvements" ? Maybe either having only one "improvement" title or specifying the type of improvements on "Additions and Improvements" title.

File: CHANGELOG.md (hunk 1 above)
Comment (MadelineMurray) [reply CHANGELOG.md]: @NicolasMassart These upcoming changes as by special request from @rojotek 

I think the rationale here is that they are breaking changes we have planned so Rob would like to give some forewarning.

File: CHANGELOG.md (hunk 2 above)
Comment (MadelineMurray) [reply CHANGELOG.md]: Yes - I did wonder about this while I was working on them. Also, the line between the two is a little blurry.  Unless you have a huge objection I think I'll add this comment to the wider release notes tasks I've got - at the moment, it's the same as for the other releases.

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,110 @@
...
+- Added the Command Line Style Guide  [\#530](https://github.com/PegaSysEng/pantheon/pull/530)
+- Documentation updates include:
+  * Migrated to new [documentation site](https://docs.pantheon.pegasys.tech/en/latest/)  
+  * Added [configuration file content](https://docs.pantheon.pegasys.tech/en/latest/Configuring-Pantheon/Using-Configuration-File/)
+  * Added [tutorial to create private network](https://docs.pantheon.pegasys.tech/en/latest/Tutorials/Create-Private-Network/)
+  * Added content on [enabling non-default APIs](https://docs.pantheon.pegasys.tech/en/latest/Reference/JSON-RPC-API-Methods/)
+  
+## Technical Improvements
+
+-  Updated `--bootnodes` command option to take zero arguments [\#548](https://github.com/PegaSysEng/pantheon/pull/527)
```
Comment (macfarla) [inline CHANGELOG.md]: 548 vs 527 ?

File: CHANGELOG.md (hunk 2 above)
Comment (arash009) [inline CHANGELOG.md]: Should we move the 0.9 changes to the top or bottom, feels a bit confusing to me what is part of 0.8.4 and what is upcoming (do we even share 0.9 items in a changelog?)

File: CHANGELOG.md (hunk 3 above)
Comment (MadelineMurray) [reply CHANGELOG.md]: Fixed

File: CHANGELOG.md (hunk 2 above)
Comment (MadelineMurray) [reply CHANGELOG.md]: Good point. Done

Comment (MadelineMurray) [reply CHANGELOG.md]: They're in the changelog at Rob's request - this appears controversial. Will need to discuss for future releases.

**Pull request merged**